platform-includes/getting-started-config/javascript.astro.mdx

```diff
@@ -18,4 +24,57 @@ export default defineConfig({
 });
 ```
 
-Once you've added your `dsn`, the SDK will automatically capture and send errors and performance events to Sentry.
+<Alert level="warning">
+  Passing runtime-specific configuration options to the Sentry integration will be deprecated in future versions.
+  We recommend passing your configuration directly to the respective `Sentry.init()` calls in `sentry.client.config.js` and `sentry.server.config.js` instead.
```

Comment:
I would be specific in saying we will no longer allow the  `dsn`, `environment`, `sampleRate`, `tracesSampleRate`, `replaysSessionSampleRate`, `replaysOnErrorSampleRate` options.

Reply:
we can also deprecate `release` since it is not used if there is a config file

Reply:
Release is actually conventionally fine as a build-time config because it also affects source-map upload.

Reply:
Conventionally yes but I think it can be misleading since this option will be disregarded in the astro config – or maybe I'm missing something

Reply:
Oh that's right - we're not forwarding it. Interesting. Yes then we can deprecate it.